README.md

```diff
@@ -23,145 +23,165 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
+docker-compose pull
+docker-compose up -d
 ```
-These commands will create:
- 1. a `deephaven-core` directory containing the source code.
- 2. Docker images containing everything needed to launch Deephaven.
 
-### Run Deephaven Database (Python)
+These commands must be run from the deployment directory.
 
-From the `deephaven-core` directory, run
+### Monitor Logs
+
+The `-d` option to `docker-compose` causes the containers to run in the background, in detached mode. 
+This option allows you to use your shell after Docker launches the containers.
+
+Since the container is running detached, you will not see any logs. However, you can follow the logs by running:
+
+```bash
+docker-compose logs -f
 ```
-    docker-compose up
+
+This command must be run from the deployment directory.
+
+Use CTRL+C to stop monitoring the logs and return to a prompt.
+
+### Shutdown
+
+The deployment can be brought down by running:
+
+```bash
+docker-compose down
 ```
-This will start the database.  The console will fill with status and logging output.
 
-Killing the process (e.g. `Ctrl+C`) will stop Deephaven.
+This command must be run from the deployment directory.
+
+### Manage example data
+
+The [Deephaven examples repository](https://github.com/deephaven/examples) contains data sets that are useful when learning 
+to use Deephaven. These data sets are used extensively in Deephaven's documentation and are needed to run some examples.
 
-### Run Deephaven Database (Groovy)
+If you have chosen a deployment with example data, the example data sets will be downloaded. Production deployments containing 
+your own data will not need the example data sets.
 
-From the `deephaven-core` directory, run
+[Deephaven's examples repository](https://github.com/deephaven/examples) contains documentation on the available data sets. 
+Additionally, there is documentation on managing the data sets. This includes instructions on how to upgrade to the latest version.
+
+To upgrade a deployment to the latest example data, run:
+
+```bash
+docker-compose run examples download
 ```
-    docker-compose --env-file default_groovy.env up
+
+To see what other example data management commands are available, run:
+
+```bash
+docker-compose run examples
 ```
-This will start the database.  The console will fill with status and logging output.
 
-Killing the process (e.g. `Ctrl+C`) will stop Deephaven.
+These commands must be run from the deployment directory.  
+If you deployment does not have example data, these commands will fail with `ERROR: No such service`.
+
 
-### Run Deephaven IDE
+## Run Deephaven IDE
 
-Once Deephaven Database is running, you can launch a Deephaven IDE in your web browser.  Deephaven IDE allows you
+Once Deephaven is running, you can launch a Deephaven IDE in your web browser.  Deephaven IDE allows you
 to interactively analyze data and develop new analytics.
```

Comment:
extra line break

Reply:
"Interactively analyze data and develop new analytics" is basically saying the same thing twice.